### DISCUSSION TITLE
How to add resample ?

### DISCUSSION THREAD

**Discussion by btc-asset-mgr** [sbooth/SFBAudioEngine#332] [Q&A]
how to add resampe in sfbaudioengine and add witch files

**Comment by sbooth** [comment]
You can do this by adding an `AVAudioUnit` of type `kAudioUnitType_FormatConverter` between the `AudioPlayer`'s player and mixer nodes, similar to adding an equalizer, and then setting the audio unit properties appropriately.

**Reply by btc-asset-mgr** [reply]
Could you give me some example？I tried but failed.

**Reply by sbooth** [reply]
It should be almost the same code as https://github.com/sbooth/SFBAudioEngine/discussions/245#discussioncomment-6125240 except using a different `AVAudioUnit`. What does your code look like that isn't working?
